partials/specials/banners/support-video.php

```diff
@@ -1,43 +1,114 @@
-          <iframe class="youtube-player lazyload" data-src="<?php echo generate_youtube_embed_url($youtube_id); ?>" frameborder="0" allowfullscreen></iframe>
-        </div>
-      </div>
-      <div class="grid-item offset-s-0 is-s-24 offset-m-2 is-m-20 is-xl-12 flex-item-xxl-4">
-        <a href="<?php echo site_url('support/'); ?>">
-          <?php
-            foreach($copy as $key => $paragraph) {
-              if ($key === 0) {
-            ?>
-            <h3 class="font-size-13 font-weight-bold mb-4 js-fix-widows"><?php echo $paragraph; ?></h3>
-            <?php
-              } else {
-            ?>
-            <p class="font-size-11 mb-4"><?php echo $paragraph; ?></p>
-            <?php
-              }
-          ?>
-          <?php
-            }
-          ?>
-        </a>
-        <a href="<?php echo site_url('support/'); ?>" class="ui-button ui-button--white">Fund something better</a>
+    <div id="<?php echo $instance; ?>" class="support-section background-red font-color-white pt-2 pb-2 pt-s-2 pb-s-2 pl-0 pt-s-5 pb-s-5 ">
+      <div class="container">
+        <form class="support-form" action="https://donate.novaramedia.com/regular">
+          <input class="support-form__value-input" type="hidden" value="<?php echo $support_section_autovalues['default']->regular_low; ?>" name="amount" />
+          
+          <!-- Grid row for video and form -->
+          <div class="grid-row font-size-11 font-size-s-10">
+            
+            <!-- Video section -->
+            <div class="grid-item is-m-24 is-l-16 is-xl-12 is-xxl-10">
+              <div class="u-video-embed-container background-red">
+                <iframe class="youtube-player lazyload" data-src="<?php echo generate_youtube_embed_url($youtube_id); ?>" frameborder="0" allowfullscreen></iframe>
+              </div>
+            </div>
+            
+            <!-- Title and form section -->
+            <div class="support-form__inputs grid-item is-l-24 offset-xl-0 is-xxl-12">
+              
+              <!-- Copy and heading -->
+              <a href="<?php echo site_url('support/'); ?>">
+                <?php
+                  if ($heading_copy || $customHeadline) {
+                ?>
+                  <div class="margin-top-micro margin-bottom-small">
+                    <h3 class="font-size-13 font-weight-bold mb-4 mt-4 js-fix-widows">
+                      <?php 
+                        if ($customHeadline) {
+                          echo $customHeadline;
+                        } else {
+                          echo $heading_copy;
+                        }
+                      ?>
+                    </h3>
+                  </div>
+                </a>
+                <?php
+                  }
+                ?>
+                
+                <!-- Support section text or custom message -->
+                <?php
+                  if ($support_section_text || $cutomText) {
```

Comment:
i think we can refactor this as we should never have a totally empty moment. In the initial variable declarations lets check for our first option, and if not set check for our second, and if thats not set fallback to hardcoded text taken from the site as it currently is. Then in this location we dont need to validate content just echo

Reply:
I have applied same thinking to the you tube ID now that we have added a fallback ID from page-support and a hard coded value.